src/util/time.h

```diff
@@ -48,5 +48,6 @@ T GetTime();
  */
 std::string FormatISO8601DateTime(int64_t nTime);
 std::string FormatISO8601Date(int64_t nTime);
+int64_t ParseISO8601DateTime(const std::string &str); 
```

Comment:
```suggestion
int64_t ParseISO8601DateTime(const std::string& str);
```

You may remove trailing whitespace with `clang-format` by running the https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy script

Reply:
Thanks :)